tests/suites/test_suite_x509parse.function

```diff
@@ -426,13 +426,15 @@ void x509_accessor_ext_types(int ext_type, int has_ext_type)
     mbedtls_x509_crt crt;
     int expected_result = ext_type & has_ext_type;
 
+    USE_PSA_INIT();
     mbedtls_x509_crt_init(&crt);
 
     crt.ext_types = ext_type;
 
     TEST_ASSERT(mbedtls_x509_crt_has_ext_type(&crt, has_ext_type) == expected_result);
 
     mbedtls_x509_crt_free(&crt);
+    USE_PSA_DONE();
```

Comment:
Pre-existing: there should be an `exit` label before call to `crt_free()`. (Otherwise I think the test generation framework will add it at the end, which is not what we want.)

(Tests need to clean up after themselves for the benefit tools that check for resource leaks. They need to do so even after an error occurred, to avoid the initial error to be buried deep in the middle of secondary errors.)